docs/tutorial/using-multiple-themes-together.md

```diff
@@ -2,4 +2,481 @@
 title: Using Multiple Themes Together
 ---
 
-_This tutorial is coming soon. In the meantime, check out the [introduction to using multiple themes in the docs](/docs/themes/using-multiple-gatsby-themes)._
+## What this tutorial covers
+
+- Composing multiple themes into a final site using [gatsby-theme-blog](/packages/gatsby-theme-blog/), [gatsby-theme-notes](/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](https://gatsby-mdx-embed.netlify.com/) as examples
+- Component shadowing
+- Component shadowing for Theme-UI theme file
+
+## Prerequisites
+
```

Comment:
Did you mean to add a link to the reference guide in the prerequisites section as well? I added a link at the end under other resources.  My sense is that second link to "What are Gatsby themes" would lead folks to the reference guide easily if they wanted.

Reply:
Oh if you have a link to that in the other resources section, then there is no need for it here my bad. I would ask that the other changes be implemented, I think it reads better.  what are your thoughts?